docs/release_notes/v0.18.1-beta.md

```diff
@@ -0,0 +1,132 @@
+# Spice v0.18.1-beta (Sep 23, 2024)
+
+The v0.18.1-beta release enhances performance for running accelerated queries across multiple datasets, introduces configurable Task History behavior, adds support for the `mssql` data connector, and improves the reliability and error handling for the `graphql`, `mysql` and `sharepoint` data connectors. It also introduces a new `spice search` CLI command, allowing users to perform embeddings-based searches across connected datasets, and adds automatic refresh triggers for the `file` data connector when changes are detected in the source data.
```

Comment:
Did someone open a task to document `spice search`?

Reply:
I'll add a page while we are building spice